Liroy/bootstrap.sh

```diff
@@ -1,38 +1,60 @@
 #!/usr/bin/env bash
 
+if [ "$(id -un)" != "root" ]; then
+  echo "sudoing.."
+  exec sudo "$0" "$@" || echo "You do not have root permission!" && exit 1
+fi
+
 echo "Updating system"
 yum -y update
 yum -y install epel-release yum-utils
 yum-config-manager \
     --add-repo \
     https://download.docker.com/linux/centos/docker-ce.repo
 
+echo "Installing git"
+yum -y install git
```

Comment:
You may combine multiple packages in one command:
`yum install -y package1 package2 package3`